README.md

```diff
@@ -70,6 +70,7 @@ Options:
   -a, --include-empty-rows            Includes empty rows in the resulting CSV output.
   -b, --with-bom                      Includes BOM character at the beginning of the CSV.
   -p, --pretty                        Print output as a pretty table. Use only when printing to console.
+  -l, --locale                        Locale to use for numbers formatting, Defaults to no locale formatting.
```

Comment:
Would need to have a way to know what options are allowed, maybe a link to a list?